src/Apps/Order/Utils/shippingUtils.ts

```diff
@@ -154,7 +154,7 @@ export const getDefaultShippingQuoteId = (order: Shipping_order) => {
 
   const cleanQuotes = compact(shippingQuotes?.edges?.map(quote => quote?.node))
 
-  return cleanQuotes ? cleanQuotes[0].id : undefined
```

Comment:
idk what kind of magic our old relay tool was doing, but happens cleanQuotes or is an empty array or with elements, never null/undefined, so this error popped once I removed the old testing tools. So far no issues/bugs were introduced